apps/src/blockly/customBlocks/googleBlockly/spritelabBlocks.js

```diff
@@ -16,7 +16,7 @@ const INPUTS = {
 export const blocks = {
   // Creates and returns a toggle button field. This field should be
   // added to the block after other inputs have been created.
-  initializeMiniToolbox(renderToolboxBeforeStack = false) {
+  initializeMiniToolbox(miniToolboxBlocks, renderToolboxBeforeStack = false) {
```

Comment:
Can you add a comment explaining that `miniToolboxBlocks` is a backwards-compatible parameter that we are ignoring?

Reply:
done!